clients/src/main/java/org/apache/kafka/common/security/authenticator/SaslServerAuthenticator.java

```diff
@@ -305,6 +327,52 @@ private boolean flushNetOutBuffer() throws IOException {
         return netOutBuffer.completed();
     }
 
+    private void handleSaslToken(byte[] clientToken) throws SaslException, IOException {
```

Comment:
I think it throws `SaslException` in the last code block in the method, but since `SaslException` is an `IOException` anyway, I have removed it from `throws`.